src/tracing/index.js

```diff
@@ -8,6 +8,24 @@ var tracingEnabled = false;
 var automaticTracingEnabled = false;
 var config;
 
+var instrumentations = [
+  './instrumentation/bluebird',
+  './instrumentation/elasticsearch',
+  './instrumentation/express',
+  './instrumentation/fastify',
+  './instrumentation/httpClient',
+  './instrumentation/httpServer',
+  './instrumentation/ioredis',
+  './instrumentation/kafka',
+  './instrumentation/mongodb',
+  './instrumentation/mssql',
+  './instrumentation/mysql',
+  './instrumentation/pg',
+  './instrumentation/redis',
```

Comment:
I think we forgot to extend this definition of entry / exits: https://github.com/instana/nodejs-sensor/blob/master/src/tracing/cls.js#L11

(and maybe we could change it in some way so that it is easier to remember?)

Reply:
Oh, bummer. I think `kafka` is also missing. For this PR I would just add mssql there.

To improve this I would 
- remove this list and the logic to set `span.k` from `cls.js` completely and make it a responsibility of the particular instrumentations to set `span.k`, probably as a non-optional argument to `startSpan`.
- Maybe add a validation check to `startSpan` which rejects/drops if `k` is not provided.
- Add expectations for `span.k` to all instrumentation tests.

But this should be a separte PR.

Reply:
![image](https://user-images.githubusercontent.com/596443/45215595-3e06a580-b29e-11e8-99b7-b951d76108b0.png)